src/SimulateAndPlot.jl

```diff
@@ -258,7 +258,7 @@ function simulate!(m::SimulationModel{FloatType,TimeType}, algorithm=missing; me
             eh.nFullRestartEvents, m.linearEquations)
 
         # Raise an error, if simulation was not successful
-        if !(solution.retcode == :Default || solution.retcode == :Success || solution.retcode == :Terminated)
+        if !(solution.retcode in [ReturnCode.Default, ReturnCode.Success, ReturnCode.Terminated])
```

Comment:
Instead of this you should use `SciMLBase.successful_retcode(solution.retcode)`. See https://docs.sciml.ai/SciMLBase/stable/interfaces/Solutions/#retcodes

Reply:
It's always re-exported by DifferentialEquations.jl

Reply:
Thanks. Fixed this as you suggested.